### PR TITLE
Check if the server is cleaned out

### PR DESCRIPTION
**Pull request by informalict**

### CODE REVIEW

File: pkg/apis/deployment/v1/plan.go [hunk 1]
```diff
@@ -157,6 +157,8 @@ const (
 	ActionTypeBootstrapSetPassword ActionType = "BootstrapSetPassword"
 	// ActionTypeMemberPhaseUpdate updated member phase. High priority
 	ActionTypeMemberPhaseUpdate ActionType = "MemberPhaseUpdate"
+	// ActionTypeSetMemberCondition sets member condition. It is high priority action.
+	ActionTypeSetMemberCondition ActionType = "SetMemberCondition"
```
Comment (ajanikow) [inline pkg/apis/deployment/v1/plan.go:161]: Add it as HighPriority in line 52

File: pkg/apis/deployment/v2alpha1/plan.go [hunk 2]
```diff
@@ -157,6 +157,8 @@ const (
 	ActionTypeBootstrapSetPassword ActionType = "BootstrapSetPassword"
 	// ActionTypeMemberPhaseUpdate updated member phase. High priority
 	ActionTypeMemberPhaseUpdate ActionType = "MemberPhaseUpdate"
+	// ActionTypeSetMemberCondition sets member condition. It is high priority action.
+	ActionTypeSetMemberCondition ActionType = "SetMemberCondition"
```
Comment (ajanikow) [inline pkg/apis/deployment/v2alpha1/plan.go:161]: run make synchronize-...

File: pkg/deployment/reconcile/action_set_member_condition.go [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+
+type actionSetMemberCondition struct {
+	// actionImpl implement timeout and member id functions
+	actionImpl
+
+	actionEmptyCheckProgress
+}
+
+// Priority returns the high priority for the set member action.
+func (a actionSetMemberCondition) Priority() api.ActionPriority {
```
Comment (ajanikow) [inline pkg/deployment/reconcile/action_set_member_condition.go:55]: Why? Check comments

**Review by ajanikow — CHANGES_REQUESTED** [review]

File: pkg/apis/deployment/v1/plan.go (hunk 1 above)
Comment (informalict) [reply pkg/apis/deployment/v1/plan.go:161]: changed

File: pkg/apis/deployment/v2alpha1/plan.go (hunk 2 above)
Comment (informalict) [reply pkg/apis/deployment/v2alpha1/plan.go:161]: launched

File: pkg/deployment/reconcile/action_set_member_condition.go (hunk 3 above)
Comment (informalict) [reply pkg/deployment/reconcile/action_set_member_condition.go]: removed

**Review by ajanikow — APPROVED** [review]

**Pull request merged**